activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -36,11 +36,25 @@ def after_teardown # :nodoc:
       class_attribute :pre_loaded_fixtures, default: false
       class_attribute :lock_threads, default: true
       class_attribute :fixture_sets, default: {}
+      class_attribute :database_transactions_config, default: {}
 
       ActiveSupport.run_load_hooks(:active_record_fixtures, self)
     end
 
     module ClassMethods
+      # Enable or disable transactions per database. This overrides the default
+      # setting as defined by `use_transactional_tests`, which applies to all
+      # database connection pools not explicitly configured here.
+      # Passing `nil` will revert config to the default setting, as specfied by
+      # `use_transactional_tests`.
+      def set_database_transactions(name, enabled)
```

Comment:
The more I stare at it, the more I don't love this exact API (though it's definitely what I originally requested). 

The alternative is two separate methods for the true and false modes (and I think we can ignore the back-to-nil/`except` case). But I'm still not sure what we'd call them. 🤔

Reply:
@matthewd what is your preference for a change to the API?

Reply:
Let's go with:

```
def use_transactional_tests_for_database(database_name, enabled = true)
def skip_transactional_tests_for_database(database_name)
```

That gives us names that parallel `use_transactional_tests`, and trade verbosity for clarity that we're accepting a DB name [vs, say, a test name].

Simple calls get friendly `use_*(:x)` / `skip_*(:x)` DSL, but if a caller is using an expression to calculate their desired state, they can just pass the boolean in to `use_*` instead of having to conditionally decide which to call.

(Big thanks @fxn for thinking this through with me!)

Reply:
And to confirm, what should `skip_transactional_tests_for_database(name)` do? Should it just call through to `use_transactional_tests_for_database(name, false)`?